identifiers/country-us/us_state_courts.csv

```diff
@@ -1,1642 +1,1682 @@
+ocd-division/country:us/state:ar/court_of_appeals:7,Arkansas Appellate Court District 7,,,
+ocd-division/country:us/state:az/court_of_appeals:1,AZ State Court of Appeals - 1st Division,,,
+ocd-division/country:us/state:az/court_of_appeals:2,AZ State Court of Appeals - 2nd Division,,,
+ocd-division/country:us/state:ca/court_of_appeals:1,CA State Court of Appeals - 1st District,,,
+ocd-division/country:us/state:ca/court_of_appeals:2,CA State Court of Appeals - 2nd District,,,
+ocd-division/country:us/state:ca/court_of_appeals:3,CA State Court of Appeals - 3rd District,,,
+ocd-division/country:us/state:ca/court_of_appeals:4,CA State Court of Appeals - 4th District,,,
+ocd-division/country:us/state:ca/court_of_appeals:5,CA State Court of Appeals - 5th District,,,
+ocd-division/country:us/state:ca/court_of_appeals:6,CA State Court of Appeals - 6th District,,,
+ocd-division/country:us/state:co/district_court:1,CO State District Court - 1st District,,,
+ocd-division/country:us/state:co/district_court:2,CO State District Court - 2nd District,,,
+ocd-division/country:us/state:co/district_court:3,CO State District Court - 3rd District,,,
+ocd-division/country:us/state:co/district_court:4,CO State District Court - 4th District,,,
+ocd-division/country:us/state:co/district_court:5,CO State District Court - 5th District,,,
+ocd-division/country:us/state:co/district_court:6,CO State District Court - 6th District,,,
+ocd-division/country:us/state:co/district_court:7,CO State District Court - 7th District,,,
+ocd-division/country:us/state:co/district_court:8,CO State District Court - 8th District,,,
+ocd-division/country:us/state:co/district_court:9,CO State District Court - 9th District,,,
+ocd-division/country:us/state:co/district_court:10,CO State District Court - 10th District,,,
+ocd-division/country:us/state:co/district_court:11,CO State District Court - 11th District,,,
+ocd-division/country:us/state:co/district_court:12,CO State District Court - 12th District,,,
+ocd-division/country:us/state:co/district_court:13,CO State District Court - 13th District,,,
+ocd-division/country:us/state:co/district_court:14,CO State District Court - 14th District,,,
+ocd-division/country:us/state:co/district_court:15,CO State District Court - 15th District,,,
+ocd-division/country:us/state:co/district_court:16,CO State District Court - 16th District,,,
+ocd-division/country:us/state:co/district_court:17,CO State District Court - 17th District,,,
+ocd-division/country:us/state:co/district_court:18,CO State District Court - 18th District,,,
+ocd-division/country:us/state:co/district_court:19,CO State District Court - 19th District,,,
+ocd-division/country:us/state:co/district_court:20,CO State District Court - 20th District,,,
+ocd-division/country:us/state:co/district_court:21,CO State District Court - 21st District,,,
+ocd-division/country:us/state:fl/district_court:6,Florida Appellate Court District 6,,,
```

Comment:
Could you either update this name or the other names for this district or the other FL district courts to be consistent?